RoboJackets/urc-software
# Transforms

# Description
This PR does the following:
- Creates ROS2 transform tree anchored at a world frame
- Calculates position of robot `base_link` frame relative to world_frame using GPS and publishes a TF2 frame.
- Calculates waypoint position and publishes a TF2 frame.

# Testing Steps (if relevant)
1. Run bringup
2. Run the `tf.launch.py` file
3. Use the web interface to publish a waypoint/base station (optional)
4. Run `ros2 run tf2_tools view_frames`

Expectation: There should a pdf file with the tf2 tree anchored at world.

# Self Checklist
